src/.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This whole file looks like a copy from somewhere else. Which parts are really required? Why can those parts not be included in the root `.gitignore`?

Reply:
I think this is the [standard GitHub Python .gitignore](https://github.com/github/gitignore/blob/main/Python.gitignore)

Reply:
Yes, this is the default Python *.gitignore* template which usually is recommended for Python projects.

Only keeping the relevant entries might be an option, mainly concerning specific libraries which are currently unused. Nevertheless, I usually prefer to keep the default *.gitignore* file as it usually is some type of common sense.

I did not add this to the root *.gitignore* file to avoid unintended side effects for now, as the whole repository consists of two more or less independent parts, while not being clearly separated for now (see #307).

Reply:
@stweil I have reduced the `.gitignore` to the entries which actually make sense in this use case (at least from my point of view).

Do you still want me to migrate the corresponding entries to the root `.gitignore` file?